OpenRA.Mods.Common/Traits/Sellable.cs

```diff
@@ -53,9 +53,9 @@ public void Sell(Actor self)
 
 			var makeAnimation = self.TraitOrDefault<WithMakeAnimation>();
 			if (makeAnimation != null)
-				makeAnimation.Reverse(self, new Sell());
+				makeAnimation.Reverse(self, new Sell(), true);
```

Comment:
`true` is already the default value for this optional parameter, so you don't need to explicitly specify it. Or did you mean for this to be `false`?